integration/vite-basename-test.ts

```diff
@@ -287,8 +287,8 @@ test.describe("Vite base / React Router basename / express dev", async () => {
   });
 
   test("works when base and basename are different", async ({ page }) => {
-    await setup({ base: "/mybase/", basename: "/mybase/app/" });
-    await workflowDev({ page, cwd, port, basename: "/mybase/app/" });
+    await setup({ base: "/mybase/", basename: "/mybase/dashboard/" });
+    await workflowDev({ page, cwd, port, basename: "/mybase/dashboard/" });
```

Comment:
Changed this to avoid ambiguity between the `app` URL path and the `app` folder in requests in vite dev mode